openpype/settings/defaults/project_settings/traypublisher.json

```diff
@@ -346,10 +346,10 @@
         }
     },
     "publish": {
-        "CollectFrameDataFromAssetEntity": {
+        "CollectSequenceFrameData": {
             "enabled": true,
             "optional": true,
-            "active": true
+            "active": false
```

Comment:
I think this is a weird default - for 99% of the files you'd drop into the publisher why wouldn't you want the frame range you're dropping in to be the frame data?

If I drop in files:
```
seq.001.exr, seq.002.exr, seq.003.exr, seq.004.exr
```
I'd expect seq to be frame 1-4 - not an arbitrary asset frame range which (might just as well be completely different, e.g. asset might be 1001-1150. Now suddenly the frame data shows 150 frames instead of the four frames the sequence actually exists of?

I feel like that'd be confusing for the majority of users as default behavior, no?
Pretty sure the default here should be `active: true` and maybe even `optional: false`?

Reply:
> This would be an issue with editorial data. This is rather safe to have it as optional.

I mean - isn't that an issue then with the actual editorial data system? :) Shouldn't that itself already provide frame ranges and if set - this plugin should not override it?

Reply:
Hope this doesn't come off as a rant but do want to highlight this:

I'm a bit worried about PRs getting merged with conversations getting marked resolved that didn't reach a valid conclusion. I want to highlight that this is already, what?, the third PR in succession covering this particular 'issue' where the fix got implemented the wrong way. It's weird that these then, especially after how much has already gone wrong with it, get merged within just a few hours span of the comments that were made on it.

It might be me being completely off here - but I really see no reason why uploading a file sequence in the tray publisher should **not** default to the frame range of the actual sequence I'm throwing into it. If it causes bugs for Editorial publishing then what we'd need to solve is those bugs and not set some odd default behavior because it's "safer". What's holding anyone from enabling this checkbox on Editorial publishing? It's just as broken if this feature is really what was causing issues. As such, it feels like this PR solved nothing except for making more odd design choices.